### ISSUE TITLE
코드 실행 관련 질문이 있습니다.

### ISSUE THREAD

**Issue by johnny9210** [JiwooKimAR/MWP-solver-with-pretrained-language-model#1]
안녕하세요 
모델을 테스트 해보려고 실행코드 내용 실행을 하면 
AttributeError: 'Dataset' object has no attribute 'netvocab2netidx' 발생하고 있는데 해결방법에 대해서 조언을 구하고 싶습니다.
감사합니다.

**Comment by JiwooKimAR** [comment]
안녕하세요. [링크](https://github.com/JiwooKimAR/Korean-MWP-dataset)에서 데이터셋을 다운 받은 후,
```
+dataset
  + chall
    + questions_train.json
    + questions_test.json
    + ...
```
와 같은 형태로 디렉토리를 변환한 후 실행해주시면 될 것 같습니다.
감사합니다.

**Comment by johnny9210** [comment]
네 답장 감사합니다.

단순한 질문일수도 있는데 제공해주신 코드는 샘플 30개 정도만 답변을 제공하는걸로 알고 있는데 train데이터 전체에 대해서 답변을
구하는 식으로 진행을 하려면 questions_train.json에 있는 텍스트 중 명사 정도만 Dataset.py의
sting_list에 추가를 하면 되는건지 궁금합니다. 혹시 추가적으로 더 진행을 해야하는 부분이 있을까요..?

2023년 6월 21일 (수) 오후 1:21, Jiwoo Kim ***@***.***>님이 작성:

**Comment by JiwooKimAR** [comment]
네 지금 Dataset.py의 string_list가 questions_train.json에 있는 여러 명사들로 이루어져 있는데, 원하신다면 더 넣으셔도 될 것 같습니다.
이후 추가적으로 진행해야 하는 사항은 없어=는 것으로 보입니다.
감사합니다.

**Comment by johnny9210** [comment]
감사합니다.

추가적인 질문이 있습니다. (LLM 관련 공부를 처음 시작해서 질문이 많은 것 양해부탁드립니다..)

readme 기준으로

python main.py --use_isc True --beam_size 1 --batch_size 32 --seed 951206
--gpu 0 --dataset 'Korean_MWPs' 으로 하면 실행이 되신다고 되어 데이터 정도만 dataset/chall
구성하고 코드를 돌릴경우
AttributeError: 'Dataset' object has no attribute 'idx2NET'. Did you mean:
'idx2IEC'? 에러가 발생하더라구요

흠 혹시 모델 학습을 하고 위의 해당 코드를 돌려야 하는걸까요? (따로 학습모델이 존재하지 않는것 같아서 질문드립니다. ㅠ)

감사합니다.

2023년 6월 27일 (화) 오후 2:00, Jiwoo Kim ***@***.***>님이 작성:

**Comment by JiwooKimAR** [comment]
안녕하세요 답변이 늦어서 죄송합니다.

--dataset chall 로 바꿔서 돌려보시면 될 것 같습니다.

그리고 main.py 는 학습과 평가를 진행하는 파일입니다.

감사합니다.
